src/ims/element/static/style.css

```diff
@@ -57,6 +57,10 @@ a.navbar-brand>img {
   flex: 2;
 }
 
+.list-group {
+  margin-bottom: 8px;
+}
```

Comment:
this was 20px before, and that was visually a lot, even for screen (and worse on print). Note this is the gray area between "LNT compliance" and "Add" on this screenshot, and the screenshot shows 8px

<img width="280" alt="image" src="https://github.com/user-attachments/assets/7e1bf680-f4cc-4a86-af90-9c4c433214d5">